modules/panther/main.nf

```diff
@@ -171,10 +192,10 @@ process PARSE_PANTHER {
     executor 'local'
 
     input:
-    tuple val(meta), val(hmmseach_json), val(epagn_tsv)
+    tuple val(meta), val(meta2a), val(hmmseach_json), val(meta2b), val(epagn_tsv)
```

Comment:
The correct way to use the `join` operator when joining on multiple keys is to pass `by: [x, y, ..., z]` where `x`, `y`, etc. are the indices to join on. Otherwise you join on the first key only so you'll end up with pairs of tuples `(meta2a, hmmseach_json)` and `(meta2b, epagn_tsv)` for unrelated sub-batches.

```suggestion
    tuple val(meta), val(meta2), val(hmmseach_json), val(epagn_tsv)
```